statusboard/src/components/PageContents/PageContents.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import About from '../../pages/About/About';
-import Projects from '../../pages/Projects/Projects';
+import Projects from '../../pages/Projects/Projects.tsx';
```

Comment:
This shouldn't need to specify `.tsx`

Reply:
I am getting

```
ESLint: Unable to resolve path to module '../../pages/Projects/Projects'.(import/no-unresolved)`.
ESLint: Unable to resolve path to module '../../pages/Projects/Projects'.(import/no-unresolved)
```

And as far as I can tell I am using the correct ESLint config file and [I believe it is based on the airbnb extension](https://stackoverflow.com/questions/59265981/typescript-eslint-missing-file-extension-ts-import-extensions)?

https://github.com/codeforamerica/brigade-project-index-statusboard/blob/9d039a4a6b85aef073522dae004f746f51526c6e/statusboard/.eslintrc.json#L2

Would you like me to revert it or change the ESLint config?